consensus/istanbul/core/core.go

```diff
@@ -353,42 +353,51 @@ func (c *core) stopTimer() {
 func (c *core) newRoundChangeTimer() {
 	c.stopTimer()
 
+	// TODO-Klaytn-Istanbul: Replace &istanbul.DefaultConfig.Timeout to c.config.Timeout
 	// set timeout based on the round number
 	timeout := time.Duration(atomic.LoadUint64(&istanbul.DefaultConfig.Timeout)) * time.Millisecond
 	round := c.current.Round().Uint64()
 	if round > 0 {
 		timeout += time.Duration(math.Pow(2, float64(round))) * time.Second
 	}
 
+	current := c.current
+	proposer := c.valSet.GetProposer()
```

Comment:
In previous code, the value is fetched when `time.AfterFunc` is executed. Can I ask why you change like this?

Reply:
@yoomee1313 `c.current` and `c.valSet.GetProposer()` is updatable in the consensus algorithm. It probably not be happened during the timer is running in the current code, but I want to make sure that timeout-related logs use the original values when it declared.